skel/problem/.gitignore

```diff
@@ -0,0 +1,3 @@
+#GENERATED BY BAPCtools
```

Comment:
I think only writing the .gitignore once `bt generate` has been invoked is preferable. That way we don't ignore things if the user doesn't use `bt generate`.